docs/platforms/react-native/tracing/instrumentation/expo-router.mdx

```diff
@@ -10,32 +10,30 @@ Sentry's React Native SDK package ships with instrumentation for Expo Router. Th
 
 The code snippet below shows how to initialize the instrumentation.
 
-```javascript { filename: 'app/_layout.tsx' }
+```javascript {6-8, 15-16, 20-25} {filename: app/_layout.tsx}
 import React from 'react';
 import { Slot, useNavigationContainerRef } from 'expo-router';
-import Constants from 'expo-constants';
+import Constants, { ExecutionEnvironment.StoreClient } from 'expo-constants';
 import * as Sentry from '@sentry/react-native';
 
-const routingInstrumentation = new Sentry.ReactNavigationInstrumentation({
-  enableTimeToInitialDisplay: Constants.appOwnership !== 'expo', // Only in native builds, not in Expo Go.
+const navigationIntegration = Sentry.reactNavigationintegration({
+  enableTimeToInitialDisplay: Constants.executionEnvironment === ExecutionEnvironment.StoreClient, // Only in native builds, not in Expo Go.
 });
 
 Sentry.init({
   dsn: '___PUBLIC_DSN___',
-  integrations: [
-    new Sentry.ReactNativeTracing({
-      routingInstrumentation,
-      enableNativeFramesTracking: Constants.appOwnership !== 'expo', // Only in native builds, not in Expo Go.
-    }),
-  ],
+  // Set tracesSampleRate to 1.0 to capture 100% of transactions for tracing.
+  // We recommend adjusting this value in production.
+  tracesSampleRate: 1.0,
+  integrations: [navigationIntegration],
+  enableNativeFramesTracking: Constants.executionEnvironment === ExecutionEnvironment.StoreClient, // Only in native builds, not in Expo Go.
 });
 
 function RootLayout() {
   const ref = useNavigationContainerRef();
-
   React.useEffect(() => {
     if (ref) {
-      routingInstrumentation.registerNavigationContainer(ref);
+      navigation.registerNavigationContainer(ref);
```

Comment:
I think you are referring to `navigationIntegration` instead of `navigation` here
```suggestion
      navigationIntegration.registerNavigationContainer(ref);
```